curriculum/challenges/english/20-upcoming-python/learn-data-structures-by-building-the-merge-sort-algorithm/656627b47bd2d2a4afbc945d.md

```diff
@@ -0,0 +1,62 @@
+---
+id: 656627b47bd2d2a4afbc945d
+title: Step 4
+challengeType: 20
+dashedName: step-4
+---
+
+# --description--
+
+In the previous step, we got the mid point. You can use it to divide `array` into two and assign each part to new variables. 
```

Comment:
Slicing is already discussed in Luhn's algorithm. Maybe this instruction can be shortened a bit.